src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/RetryExceptionPolicyFactory.cs

```diff
@@ -58,7 +58,7 @@ private static AsyncRetryPolicy CreateExtendedRetryPolicy(int maxRetries, int ma
         {
             return Policy.Handle<RequestRateExceededException>()
                 .Or<CosmosException>(e => e.IsRequestRateExceeded())
-                .Or<CosmosException>(e => e.StatusCode == System.Net.HttpStatusCode.ServiceUnavailable)
+                .Or<CosmosException>(e => (e.StatusCode == System.Net.HttpStatusCode.ServiceUnavailable || e.StatusCode == System.Net.HttpStatusCode.RequestTimeout))
```

Comment:
Could we add a unit test for this? It looks like the 503 is being tested in CosmosFhirDataStoreTests.cs, line 204.